src/Appwrite/Exception/Certificate.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Appwrite\Exception;
+
+class Certificate extends \Exception
+{
+    private $incrementAttempts = false;
+
+    // Only set incrementAttempts to TRUE if exception occured AFTER certbot command execution
+    public function __construct(string $message, bool $incrementAttempts = false)
```

Comment:
This is not a very common way for us to use an exception.

Reply:
What's the better way? I need to differentiate between expected error, and any unexpected one. Unexpected should be logged into container, while expected should be logged into database.

Reply:
I removed increment attempts parameter from exception.